cypress/e2e/Mobile/MB_SignUp_Scenarios.cy.js

```
@@ -0,0 +1,27 @@
+describe('Register New User Tests', () => {
+    const viewports = [
+      { name: 'mobile', width: 375, height: 812 }, //define the test target for test environment
+    ];
+  
+    viewports.forEach((viewport) => {
+      context(`Viewport: ${viewport.name}`, () => {
+        beforeEach(() => { //clear all cookies and storaged information before run tests
+          cy.clearCookies();
+          cy.clearLocalStorage();
+          cacheAcrossSpecs: true;
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)